portals/publisher/services/login/login_callback.jag

```diff
@@ -0,0 +1,201 @@
+<%
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@chanaka3d as we have removed jag files from portals now I think we will have to apply this to the relevant login_callback.jsp file.